docs/user-guide/filtering-layers.md

```diff
@@ -114,6 +116,14 @@ It is possible to filter records in the [Attribute Table](attributes-table.md#at
 
 The list of records in the *Attribute Table* will be automatically filtered according to such user selection and then the user can disable the geometry filter through the **Remove filter** <img src="../img/button/remove_filter_geometry.jpg" class="ms-docbutton"/> button.
 
+#### Quick Filter by viewport
+
+On the [Attribute Table](attributes-table.md#attribute-table) the user can filter data by the current viewport, to activating the  **Filter by viewport** <img src="../img/button/filter-by-viewport-button.jpg" class="ms-docbutton"/> button. Once clicked, the button turns green and the list of records in the *Attribute Table* shows which records of the layer are in the current viewport. Browsing in the map view, the records visible in the *Attribute Table* change automatically.
```

Comment:
On the [Attribute Table](attributes-table.md#attribute-table) the user can filter data by map viewport through the **Filter by viewport** <img src="../img/button/filter-by-viewport-button.jpg" class="ms-docbutton"/> button. Once clicked, the toggle button turns its state to green and the list of records in the *Attribute Table* is filtered by showing only records corresponding to layer features present in the current map viewport.